tests/test_config_image.py

```diff
@@ -1359,6 +1404,12 @@ def test_tiled():
         galsim.config.BuildImage(config)
     config['image']['stamp_ysize'] = ysize
     config['image']['yborder'] = xborder
+    with assert_raises(galsim.GalSimConfigError):
+        galsim.config.BuildImage(config)
+    config['image']['yborder'] = -yborder
+
+    # numpy types need np. or numpy. prefix.
+    config['image']['dtype'] = 'float64'
```

Comment:
Plain `float64` seems like an intuitive use of dtype here too (`str(np.array(3).dtype)` returns just the 'int64', e.g.).  Can we enable this?

Reply:
OK.  Done.